meetings/2021-11-23.md

```diff
@@ -0,0 +1,115 @@
+
+* Pavlik: Lines up with where i"m hoping to head with delegated / trusted grants. Already have setup for delegated grants between owner and application they use. Can use the same pattern in a chain. Also need to have an upstream notification of delegation. Omni gives Alice access. Alice delegates that access to Bob. Alice needs to let Omni know so it can update its records. Will share a link to authz panel. 
+
+* Discussion: https://github.com/solid/authorization-panel/discussions/271
+
+* Eric: One strategy is proxying - where Alice never needs to know that Acme delegated. Could say that anyone can submit to me a proof chain and I'll just check it, and no one needs to be notified of those delegations.
+
+* Pavlik: In this scenario, the grantee shouldn't have knowledge of what was granted, the delegation should be narrow. Upstream party should know, downstream shouldn't.
+
+* Justin: I find delegation chain approach elegant and clean. Keeping the upstream informed would need to be addressed. At least telling them where to go to get informed.
+
+* Pavlik: To complete the trusted grants we probably need to solve this. Don't need the weirdness of control access.
+
+### [Dynamic authorization by reference list](https://github.com/solid/data-interoperability-panel/issues/174)
+
+* Justin: https://hackmd.io/@hAlm_W2-T-q16baquNyRXg/HyS6Xuqdt
+* ...: It gets the concept across, I used health scenario since we deal with it a lot.
+* ...: You have Data Registry and Data Registration for medical records. Medical record is composed of prescriptions and appointments and conditions. Medical record is more of a meta object which just links to all those other things.
+* ...: If I want to share the whole medical record with someone who helps taking care of me, I can share the whole medical record with them. To give partial access, in that case Alice wants to use Prescripto application. She wants it to access prescriptions but not conditions, appointments etc. Prescripto needs to see all the prescriptions and also if it adds new one it needs to link it to medical record. For prescripto to know it needs to be able to see those links. If Medical Record contains links to everything (conditions, appointments, prescriptions). Even that Prescripto can't follow those links, it still leaks data by knowing how many conditions and appointments there are linked. Which on itself is not acceptable.
+* ...: We either need to say that Prescrito gets partial access to medical record, which would require something like shapemasks or tripple based authorization. Or we need to maintain those links in stand alone resources. This term of reference lists comes from that. We could say that Prescripto has access to prescriptions reference list but not to reference lists of conditions and appointments. It seems like the most straight forward approach. We would need to ensure that shape paths can follow external resources.
```

Comment:
```suggestion
* Justin: https://hackmd.io/@hAlm_W2-T-q16baquNyRXg/HyS6Xuqdt
  * It gets the concept across. I used health scenario, since we deal with it a lot.
  * You have Data Registry and Data Registration for medical records. Medical record is composed of prescriptions and appointments and conditions. Medical record is more of a meta-object which just links to all those other things.
  * If I want to share the whole medical record with someone who helps taking care of me, I can share the whole medical record with them. To give partial access, such as when Alice wants to use Prescripto application, she wants it to access prescriptions, but not conditions, appointments, etc. Prescripto needs to see all the prescriptions, and also if it adds new one, it needs to link it to medical record. For Prescripto to know, it needs to be able to see those links. If Medical Record contains links to everything (conditions, appointments, prescriptions). Even if Prescripto can't follow those links, it still leaks data by knowing how many conditions and appointments are linked, which in itself is not acceptable.
  * We either need to say that Prescripto gets partial access to medical record, which would require something like shapemasks or triple-based authorization, or we need to maintain those links in standalone resources. The "reference lists" term comes from that. We could say that Prescripto has access to prescriptions reference list, but not to reference lists of conditions and appointments. It seems like the most straight forward approach. We would need to ensure that shape paths can follow external resources.
```